README.md

```diff
@@ -199,6 +199,40 @@ reload your webserver without having permission to do anything else.
 * Don't allow this script to be able to read your domain private key!
 * Don't allow this script to be run as root!
 
+### Example user setup
+
+```
+# create 'acme' user which will run the script
+useradd acme -d /etc/ssl/acme/ -m
+
+# challenges folder writable for the script, readable for the web server
+mkdir /var/www/challenges/
+chown acme:www-data /var/www/challenges/
+chmod 750 /var/www/challenges/
+
+# create an account key readable only by the script
+openssl genrsa 4096 > /etc/ssl/acme/account.key
+chown acme:acme /etc/ssl/acme/account.key
+chmod 400 /etc/ssl/acme/account.key
+
+# create domain key readable only by the web server
+openssl genrsa 4096 > /etc/ssl/acme/domain.key
+chown www-data:www-data /etc/ssl/acme/domain.key
+chmod 400 /etc/ssl/acme/domain.key
```

Comment:
Thanks for the tips, I'll incorporate them in the readme. As for the web server directory permissions I usually only allow read and write permissions where strictly needed, but perhaps this is a bit overkill.